src/core/Akka.Tests.Performance/Actor/AddressSpec.cs

```diff
@@ -33,6 +34,7 @@ public void ParseLocalThroughput(BenchmarkContext context)
         public void ParseRemoteThroughput(BenchmarkContext context)
         {
             Address.Parse("akka://sys");
+            _parseThroughput.Increment();
```

Comment:
Calls the NBench `Counter` object that's used to measure the throughput of the `Address.Parse` function, which I omitted by accident before.